<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UProgUpdaterDialog</class>
 <widget class="QDialog" name="UProgUpdaterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поновлення програми податкова звітність</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/UAutoParkImg.qrc">
    <normaloff>:/Program/small/bus-32x32.png</normaloff>:/Program/small/bus-32x32.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="labelFormInfo">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>НЕОБХІДНЕ ПОНОВЛЕННЯ ПРОГРАМИ &quot;АВТОПАРК МЕНЕДЖЕР&quot;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Інформація про версії</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="labelCurVersion">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelNewVersion">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Процес поновлення</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="labelCurOperationInfo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelProgressInfo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButtonUpdate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="text">
        <string>Поновити (автоматично)</string>
       </property>
       <property name="icon">
        <iconset resource="../images/UAutoParkImg.qrc">
         <normaloff>:/function_icon_set/function_icon_set/refresh_48.png</normaloff>:/function_icon_set/function_icon_set/refresh_48.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Скасувати і ввійти</string>
       </property>
       <property name="icon">
        <iconset resource="../images/UAutoParkImg.qrc">
         <normaloff>:/function_icon_set/function_icon_set/cross_48.png</normaloff>:/function_icon_set/function_icon_set/cross_48.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/UAutoParkImg.qrc"/>
 </resources>
 <connections/>
</ui>
